print_DeepTauIdInput.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/RecoTauTag/RecoTau/plugins/DeepTauId.cc b/RecoTauTag/RecoTau/plugins/DeepTauId.cc
+index 1c2534f3784..f34623b990a 100644
+--- a/RecoTauTag/RecoTau/plugins/DeepTauId.cc
++++ b/RecoTauTag/RecoTau/plugins/DeepTauId.cc
+@@ -1160,7 +1160,7 @@ public:
+     desc.add<bool>("disable_dxy_pca", false);
+     desc.add<bool>("disable_hcalFraction_workaround", false);
+     desc.add<bool>("disable_CellIndex_workaround", false);
+-    desc.add<bool>("save_inputs", false);
++    desc.add<bool>("save_inputs", true);
```

Comment:
don't change default values, set `save_inputs` in the producer config when rerunning tau ids

Reply:
But is it using `edm::InputTag()` for these values? Because I thought it is just hard-coded state, that can not be changed on the config level